server/bleep/src/webserver/answer.rs

```diff
@@ -337,7 +335,15 @@ impl<'a> AnswerAPIClient<'a> {
             A:",
             snippet.relative_path, snippet.text, self.query
         );
-        let tokens_used = self.semantic.gpt2_token_count(&prompt);
+        prompt
+    }
+
+    async fn select_snippet(&self, prompt: &str) -> Result<reqwest::Response, reqwest::Error> {
+        self.send(prompt.to_string(), 1).await
```

Comment:
just a thought, `AnswerAPIClient::send` should accept a `&str` and the call site needn't pass owned data, not important to put in this PR.